tests/unit/neptune/new/test_stringify_unsupported.py

```diff
@@ -439,8 +439,8 @@ def test_extend__dict(self, run):
 
     def test_append__float(self, run):
         with assert_no_warnings():
-            run["stringified"].append(stringify_unsupported(1.0))
```

Comment:
I think it'd be great to test both `append()` with and without the `step` argument. WDYT?

Reply:
Yes, I was thinking about doing it but I was definitely too lazy :P. Give me a minute.